doc/doxygen/Makefile

```diff
@@ -10,13 +10,23 @@ doc: html
 
 # by marking html as phony we force make to re-run Doxygen even if the directory exists.
 .PHONY: html
-html:
+html: src/css/riot.css
 	( cat riot.doxyfile ; echo "GENERATE_HTML = yes" ) | doxygen -
 
 .PHONY: man
 man:
 	( cat riot.doxyfile ; echo "GENERATE_MAN = yes" ) | doxygen -
 
+ifneq (,$(shell which lessc))
```

Comment:
Yapp, but this wouldn't be changed by this warning ;-). Only the annoyance of some users.